features/iotserver-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/store/extensions/app/store-device-type/apis/device-api.jag

```diff
@@ -38,7 +38,9 @@ var user = server.current(session);
 var result;
 
 if (!user) {
-	response = responseProcessor.buildErrorResponse(response, 401, "Unauthorized");
+	//response = responseProcessor.buildErrorResponse(response, 401, "Unauthorized");
```

Comment:
We have dropped ES extended approach and moved code into [[1]](https://github.com/wso2/carbon-device-mgt-plugins/tree/master/components/iot-plugins). This file is does not exist in latest repo Hence cannot fix.